tools/cassandra_delete_range/internal/cass/cass.go

```diff
@@ -180,100 +189,193 @@ func (c *ClioCass) pruneData(
 
 	// successor queries
 	if !c.settings.SkipSuccessorTable {
+		file, err := os.Create("continue.txt")
```

Comment:
We might be able to tidy up all logic about "continue.txt" to a class. Then you can get rid of hardcoded file name and repeated logic. You can also leverage the object scope to manage the file creation and deletion.